### PR TITLE
Fix typos in paper and docstrings

### PR DESCRIPTION
**Pull request by pitmonticone**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -17,7 +17,7 @@
 
 ## API changes
 - `SurrogateGenerator`s now have a field `s` into which surrogates are generated, avoiding
-    unneccessary memory allocations. The
+    unnecessary memory allocations. The
```
Comment (kahaaga) [inline CHANGELOG.md:20]: Good catch, but the correction is also not correct. This should be ["unnecessary"](https://www.oed.com/dictionary/unnecessary_adj?tab=factsheet#16604305) (with two n-s and two s-s).

**Review by kahaaga — CHANGES_REQUESTED** [review]
Thanks for the fixes! This looks good to me - after my one comment has been addressed.

File: CHANGELOG.md (hunk 1 above)
Comment (pitmonticone) [reply CHANGELOG.md:20]: Ops, I'm sorry. You're perfectly right.

Comment (kahaaga) [reply CHANGELOG.md:20]: > Hi @kahaaga, I'm not sure I understand. Isn't that the correction I've pushed?

You are absolutely correct. It appears I'm the jet-lagged one. Forget my comment entirely!

**Pull request merged**